lib/active_shipping/carriers/ups.rb

```diff
@@ -411,51 +452,35 @@ def build_delivery_dates_request(origin, destination, packages, pickup_date, opt
       xml_builder.to_xml
     end
 
-    def build_international_shipment_request_options(xml, origin, destination, packages, options)
-      build_location_node(xml, 'SoldTo', options[:sold_to] || destination, options)
+    def build_international_forms(xml, origin, destination, packages, options)
       if options[:paperless_invoice]
-        xml.ShipmentServiceOptions do
-          xml.InternationalForms do
-            xml.FormType('01') # 01 is "Invoice"
-            xml.InvoiceDate(options[:invoice_date] || Date.today.strftime('%Y%m%d'))
-            xml.ReasonForExport(options[:reason_for_export] || 'SALE')
-            xml.CurrencyCode(options[:currency_code] || 'USD')
-
-            if options[:terms_of_shipment]
-              xml.TermsOfShipment(options[:terms_of_shipment])
-            end
+        xml.InternationalForms do
+          xml.FormType('01') # 01 is "Invoice"
+          xml.InvoiceDate(options[:invoice_date] || Date.today.strftime('%Y%m%d'))
+          xml.ReasonForExport(options[:reason_for_export] || 'SALE')
+          xml.CurrencyCode(options[:currency_code] || 'USD')
+
+          if options[:terms_of_shipment]
+            xml.TermsOfShipment(options[:terms_of_shipment])
+          end
 
-            packages.each do |package|
-              xml.Product do |xml|
-                xml.Description(package.options[:description])
-                xml.CommodityCode(package.options[:commodity_code])
-                xml.OriginCountryCode(origin.country_code(:alpha2))
-                xml.Unit do |xml|
-                  xml.Value(package.value / (package.options[:item_count] || 1))
-                  xml.Number((package.options[:item_count] || 1))
-                  xml.UnitOfMeasurement do |xml|
-                    # NMB = number. You can specify units in barrels, boxes, etc. Codes are in the api docs.
-                    xml.Code(package.options[:unit_of_item_count] || 'NMB')
-                  end
+          packages.each do |package|
+            xml.Product do |xml|
+              xml.Description(package.options[:description])
+              xml.CommodityCode(package.options[:commodity_code])
```

Comment:
Is `CommodityCode` the harmonized system code? If so, can we call it that in the options hash?